scripts/build-toolchains.sh

```diff
@@ -134,18 +134,16 @@ SRCDIR="$(pwd)/toolchains" module_all qemu --prefix="${RISCV}" --target-list=ris
 git submodule update --init $CHIPYARD_DIR/tools/dromajo/dromajo-src
 make -C $CHIPYARD_DIR/tools/dromajo/dromajo-src/src
 
-cd "$RDIR"
-
 # create specific env.sh
-{
-    echo "# auto-generated by build-toolchains.sh"
-    echo "export CHIPYARD_TOOLCHAIN_SOURCED=1"
-    echo "export RISCV=$(printf '%q' "$RISCV")"
-    echo "export PATH=\${RISCV}/bin:\${PATH}"
-    echo "export LD_LIBRARY_PATH=\${RISCV}/lib\${LD_LIBRARY_PATH:+":\${LD_LIBRARY_PATH}"}"
-} > env-$TOOLCHAIN.sh
+cat > "$CHIPYARD_DIR/env-$TOOLCHAIN.sh" <<EOF
```

Comment:
Since this changes the toolchain script, can we change the version of the tools (`tools-cache-version`) in `.circleci/config.yml` so that the toolchain is rebuilt.

Reply:
done